v0/destinations/monday/transform.js

```diff
@@ -0,0 +1,124 @@
+const get = require("get-value");
+const { EventType } = require("../../../constants");
+const { ENDPOINT } = require("./config");
+const { populatePayload, getBoardDetails } = require("./util");
+const {
+  defaultRequestConfig,
+  defaultPostRequestConfig,
+  removeUndefinedAndNullValues,
+  CustomError,
+  getErrorRespEvents,
+  getSuccessRespEvents,
+  getDestinationExternalID
+} = require("../../util");
+
+const responseBuilder = (payload, endpoint, apiToken) => {
+  if (payload) {
+    const response = defaultRequestConfig();
+    response.endpoint = endpoint;
+    response.headers = {
+      "Content-Type": "application/json",
+      Authorization: `${apiToken}`
+    };
+    response.method = defaultPostRequestConfig.requestMethod;
+    response.body.JSON = removeUndefinedAndNullValues(payload);
+    return response;
+  }
+  throw new CustomError(
+    "Monday]: Payload could not be populated due to wrong input",
+    400
+  );
+};
+
+/**
+ * This function is used to build the response for track call.
+ * @param {*} message
+ * @param {*} param1
+ * @returns
+ */
+const trackResponseBuilder = async (message, { Config }) => {
+  const { apiToken } = Config;
+  let boardId = getDestinationExternalID(message, "boardId");
+  if (!boardId) {
+    boardId = Config.boardId;
+  }
+  if (!boardId) {
+    throw new CustomError("Monday]: boardId is a required field", 400);
+  }
+  const event = get(message, "event");
+  const endpoint = ENDPOINT;
+  if (!event) {
+    throw new CustomError(
+      "[Monday]: event is not present in the input payloads",
+      400
+    );
+  }
+
```

Comment:
We have to take a list of events from the dashboard for which we will proceed. No need to process it for all of the events.

Reply:
Added it